posts/2015-07-25-new-javascript-generators.md

```diff
@@ -0,0 +1,120 @@
+---
+title: New JavaScript Generators
+author: Freezeboy
+date: 2015-07-25 00:00
+toc: true
+---
+
+# Summary
+
+With the upcoming Servant version, it will be way easier to build a javascript
```

Comment:
Why not be more descriptive of the actual changes? Maybe something like

> The upcoming version of *servant* will give you much more control over the style of javascript client libraries you can generate. [...]

Also, take credit for the change (i.e. - say "I've worked on ..." or whatever)!